akka-actor-typed/src/main/scala/akka/actor/typed/Behavior.scala

```diff
@@ -252,6 +262,8 @@ object Behavior {
   private[akka] class StoppedBehavior[T](val postStop: OptionVal[Behavior[T]]) extends Behavior[T] {
     validatePostStop(postStop)
 
+    def cast[U]: Behavior[U] = unsafeCast[U] // won't receive more messages so cast is safe
```

Comment:
why is this needed? why not use unsafeCast directly?

Reply:
Not strictly needed.  Can be just unsafeCast with the justifying comment, instead.